### ISSUE TITLE
Flow v57.1  can't determine EventType for evt.target, Flow v56 worked fine... 

### ISSUE THREAD

**Issue by lvdang** [facebook/flow#5102]
If this is a bug, please try reproducing using https://flow.org/try/.

Clarification:  evt: Event. "evt is an Event type". I looked in the dom file since Node extends EventType, don't know why evt.target will fail.

264:     if (this.trigger && this.state.open && (evt.target === this.trigger || this.trigger.contains(evt.target))) {
                                                                                                      ^^^^^^^^^^ EventTarget. This type is incompatible with the expected param type of
438:   contains(other: ?Node): boolean;
                        ^^^^ Node. See lib: /private/tmp/flow/flowlib_2eac43bc/dom.js:438

**Comment by CharlesStover** [comment]
Extending EventType doesn't make it an EventType.

An Employee is a Person, but a Person is not necessarily an Employee. Employee extends Person, but you cannot pass a Person as an argument to a function expecting an Employee. In this case, the function appears to be expecting a Node, and even though Node extends EventType (as Employee extends Person), you cannot pass it an EventType (as you could not pass a Person to a parameter expecting an Employee).

**Comment by lvdang** [comment]
Got it - thanks

**Comment by apsavin** [comment]
@CharlesStover that's true, but I think it's pretty safe to pass `EventTarget` into `contains`, so should be fixed.

Maybe we should change the type of `contains` to `contains(other: ?Node | ?EventTarget)`

**Comment by SamChou19815** [comment]
not enough info to repro
